### ISSUE TITLE
Add weight to KL loss in VariationalAutoencoder.java implementation

### ISSUE THREAD

**Issue by shaulz** [eclipse/deeplearning4j#6818]
In Keras examples https://github.com/fchollet/deep-learning-with-python-notebooks/blob/master/8.4-generating-images-with-vaes.ipynb, the KL loss is assigned with a small weight.

Please add weight for KL loss as a parameter for VariationalAutoencoder configuration.

In the following extract from the Keras code the KL weight is -5e-4.

```
def vae_loss(self, x, z_decoded):
    x = K.flatten(x)
    z_decoded = K.flatten(z_decoded)
    xent_loss = keras.metrics.binary_crossentropy(x, z_decoded)
    kl_loss = -5e-4 * K.mean(1 + z_log_var - K.square(z_mean) - K.exp(z_log_var), axis=-1)
return K.mean(xent_loss + kl_loss)
```

Aha! Link: https://skymindai.aha.io/features/DL4J-43

**Comment by agibsonccc** [comment]
Closing — this issue is over 7 years old with no recent activity. The codebase has undergone a major rewrite. If this is still relevant, please open a new issue with current reproduction steps.
